Makefile

```diff
@@ -86,6 +146,19 @@ else
 TESTCMD = -run $(RUNTEST)
 endif
 
+BLOCKLIST='/tests/'
+COVERAGE_FILE='coverage.txt'
+
+coverage: build-all kind
+	PATH="${PWD}:${PATH}" go test $$(go list ./... | grep -v $(BLOCKLIST)) \
```

Comment:
the indentation you picked in this PR dosent match the rest of the makefile, is their a reason for this?